testsuites/CBLTester/topology_specific_tests/multiple_sync_gateways/conftest.py

```diff
@@ -248,6 +251,12 @@ def params_from_base_suite_setup(request):
         expected_server_version=server_version,
         expected_sync_gateway_version=sync_gateway_version
     )
+
+    need_sgw_admin_auth = (not disable_admin_auth) and sync_gateway_version >= "3.0"
```

Comment:
Cleanup prints

Reply:
cleaned up the print statement, and added proper log_info